docs/meshstack.azure.index.md

```diff
@@ -36,10 +36,8 @@ In most organizations, other applications like Office 365 already consume user i
 
 ### Workload AAD Tenant
 
-Because meshcloud requires read-write permissions to manage user roles on Azure Subscriptions, we recommend creating a
-separate "Workload AAD Tenant" to be exclusively used by meshcloud. Our orchestration engine then creates Guest Users in the meshcloud AAD Tenant that reference user identities from the "home tenant" (AAD B2B). This way, users have a single cloud identity managed
-by your organization-wide policies while isolating "workload" related Azure activies into its own AAD Tenant which has no
-way of affecting other applications using the home tenant like Office 356 etc.
+Because meshcloud requires read-write permissions to your Active Directory to manage user roles on Azure Subscriptions, we recommend creating a
```

Comment:
```suggestion
Because meshcloud requires read-write permissions to your Azure Active Directory to manage user roles on Azure Subscriptions, we recommend creating a
```

Reply:
fixed